README.md

```diff
@@ -1,6 +1,5 @@
 <img align="right" src=".assets/app-icon-rounded.png" width="70" alt="Contribution Graphs for GitHub">
 
-# GitHub Contribution Graphs for iOS
```

Comment:
Hi @abdullah818, thanks for your contribution, what's the reason for removing the README title? 🤔